### PR TITLE
[436] Keep metrics for more than 1 month

### PR DESCRIPTION
**Pull request by saliceti**
## What
Create a _cold_ prometheus reading from the long term retention policy

## How to review
- Check billing dashboard in BAT QA for more than 30 days: https://grafana-bat-qa.london.cloudapps.digital/d/tIw8h_G7z/paas-billing?orgId=1&refresh=1h
- Deploy _prometheus cold_ in your own monitoring by following the Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,27 @@ The git reference can be changed. For example for the `dev` branch:
 source = "git::https://github.com/DFE-Digital/cf-monitoring.git//prometheus_all?ref=dev"
 ```
 
+## Retention policy
+The default retention policy in influxdb is 30 days. After which all the metrics are deleted. It is possible to keep some metrics for 12 months using
+[influxdb downscaling](https://docs.influxdata.com/influxdb/v1.8/guides/downsample_and_retain/) and enable _yearly_ prometheus.
```
Comment (NickGraham101) [inline README.md:58]: `influxdb downsampling` rather than `influxdb downscaling`

File: README.md [hunk 2]
```diff
@@ -53,6 +53,27 @@ The git reference can be changed. For example for the `dev` branch:
...
+
+### Influxdb downscaling
```
Comment (NickGraham101) [inline README.md:60]: `Influxdb downsampling` rather than `Influxdb downscaling`

Comment (saliceti) [reply README.md]: Thanks!

**Review by NickGraham101 — APPROVED** [review]

**Comment by saliceti** [comment]
[Billing dashboard: keep metrics for more than 1 month](https://trello.com/c/pisgYfDl/436-billing-dashboard-keep-metrics-for-more-than-1-month)

**Review by felixtheflex — APPROVED** [review]

**Pull request merged**